### PR TITLE
[5.0] Outliner: We don't support calls with opened existentials

### PR DESCRIPTION
**Pull request by aschwaighofer**
rdar://47099026

### CODE REVIEW

**Comment by aschwaighofer** [comment]
@swift-ci Please test

File: test/SILOptimizer/outliner.swift [hunk 1]
```diff
@@ -175,3 +175,6 @@ extension Operation {
   }
 }
 
+public func dontCrash(a: Any)  {
```
Comment (slavapestov) [inline test/SILOptimizer/outliner.swift]: Can you pattern match the SIL to show that there's actually an opened type in there, or even write a SIL test case?

**Comment by aschwaighofer** [comment]
@swift-ci Please test

**Review by eeckstein — APPROVED** [review]
lgtm

**Pull request merged**